main.js

```diff
@@ -30,40 +22,24 @@ class InitLayer {
 // lancement des web worker pour remonter les datas
 
 const giveData = new Worker("info.js");
+let federations = {};
```

Comment:
I turned `federations` into an object that will contain `InitLayer`s classified by federation. Maybe the name is therefore not appropriate.
The end will be something like:
```
federations: {
  "BA": [InitLayer, InitLayer, InitLayer,…],
  "ON": [InitLayer, InitLayer, InitLayer, InitLayer,…],
  …
}
```